pkg/reconciler/apis/apiexport/apiexport_controller.go

```diff
@@ -230,6 +261,45 @@ func (c *controller) enqueueSecret(obj interface{}) {
 	}
 }
 
+func (c *controller) enqueueFromAPIBinding(obj interface{}) {
+	binding, ok := obj.(*apisv1alpha1.APIBinding)
+	if !ok {
+		return
+	}
+
+	// Skip any bindings that haven't progressed to initially bound.
+	cond := conditions.Get(binding, apisv1alpha1.InitialBindingCompleted)
+	if cond != nil && cond.Status != corev1.ConditionTrue {
+		return
+	}
+
+	logger := logging.WithObject(logging.WithReconciler(klog.Background(), controllerName), binding)
+
+	apiExports, err := c.apiExportIndexer.ByIndex(indexers.APIExportByClusterAndName,
+		indexers.ClusterPathAndAPIExportName(binding.Spec.Reference.Workspace.Path, binding.Spec.Reference.Workspace.ExportName),
```

Comment:
I imagine we also want this closed over as a func in the controller, for mocking, and expect the workspace ref as input?

Reply:
Will fix